ui-tests/src/test/java/io/syndesis/qe/pages/customizations/api_client_connectors/detail/ApiClientConnectorDetail.java

```diff
@@ -0,0 +1,52 @@
+package io.syndesis.qe.pages.customizations.api_client_connectors.detail;
+
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selectors.byId;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+public class ApiClientConnectorDetail extends SyndesisPageObject {
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-api-connector-detail");
+	}
+
+	@Override
+	public SelenideElement getRootElement() {
+		return $(Element.ROOT).should(Condition.exist);
+	}
+
+	@Override
+	public boolean validate() {
+		try {
+			$(Element.ROOT).should(exist);
+			return true;
+		} catch (Throwable t) {
+			System.out.println("THROWABLE: " + t.getClass().getName());
```

Comment:
Pls use standard lombok log here. But I would rather do something like `return getRootElement().exists();` across all of those `validate()` methods.

Reply:
this was forgotten println. I removed it. The method returns false if the validation is unsuccessful.